downstream/modules/platform/con-aap-installation-on-disconnected-rhel.adoc

```diff
@@ -1,34 +1,34 @@
-
+// Module included in the following assemblies:
+// assembly-disconnected-installation.adoc
 
 [id="con-aap-installation-on-disconnected-rhel_{context}"]
 
-
-= {PlatformNameShort} installation on disconnected RHEL
+= Installing {PlatformNameShort} on disconnected RHEL
 
 [role="_abstract"]
-Install Ansible Automation Platform (AAP) automation controller and a private automation hub, with an installer-managed database located on the automation controller without an Internet connection.
+You can install {PlatformNameShort} {ControllerName} and a {PrivateHubName} without an internet connection using the installer-managed database located on the {ControllerName}.
 
 == Prerequisites
 
-To install AAP on a disconnected network, complete the following prerequisites:
+To install {PlatformNameShort} on a disconnected network, complete the following prerequisites:
```

Comment:
Per our discussion earlier today, we'll need to either reorganize the modules and create a task/procedure module that these prereqs would fit into. And once we do that, we should ensure that the prereqs comply with modularization standards [here](https://redhat-documentation.github.io/modular-docs/#con-creating-procedure-modules_writing-mod-docs), specifically under sub-heading, "Procedure prerequisites", or in the Red Hat Supplementary Guide in the [Prerequisites](https://redhat-documentation.github.io/supplementary-style-guide/#prerequisites) section.

Reply:
I ended up putting these prerequisites in the assembly module--according to our [mod documentation guide](https://redhat-documentation.github.io/modular-docs/#forming-assemblies), assemblies can include prerequisites. (This may change further down the line when I break all this content out into three separate assemblies)